api/app/schemas/hfi_calc.py

```diff
@@ -191,8 +191,10 @@ class HFIAdminRemovedStation(BaseModel):
 
 class HFIAdminStationUpdateRequest(BaseModel):
     """ Request input for updating a batch of HFI stations. """
+    fire_centre_id: int
     added: List[HFIAdminAddedStation]
     removed: List[HFIAdminRemovedStation]
+    date_range: Optional[DateRange]
```

Comment:
Maybe I'm missing it - but where is this date range being used?

The only place I can find is in unready_planning_areas, and there, only the start_date of the range is being used.